extensions/amp-lightbox-gallery/1.0/storybook/Basic.js

```diff
@@ -8,6 +8,12 @@ import {
   WithBentoLightboxGallery,
 } from '../component';
 
+// Workaround to import css for React Storybook
+import '../component.jss';
+
+// Workaround to import css for React Storybook
```

Comment:
These comments may be lost/shifted when eslint auto-sorts+groups imports

Reply:
Hm...should i just remove the comments exampling them?